### PR TITLE
Prepare for black

### PR DESCRIPTION
**Pull request by inglesp**
Prepare for using black by removing various constructions that are syntax errors in Python 3.

### CODE REVIEW

**Comment by evansd** [comment]
Should we replace codeclimate with a travis job that runs `black --check` (probably as an allowed failure)?

File: openprescribing/prints.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+./common/generate_measure_sql.py
```
Comment (evansd) [inline openprescribing/prints.txt]: Did you intend to commit this file?

Comment (inglesp) [reply openprescribing/prints.txt]: I think you know the answer to that!

**Comment by sebbacon** [comment]
In the past I've found codeclimate's code complexity warnings to be useful pointers to code smells - but I think we've (at least in effect?) got that all turned off anyway, because they're only useful sometimes.  So I think I'd be happy with replacing at least the python parts of codeclimate with a black test

**Pull request merged**